src/components/navigation/navigation.tsx

```diff
@@ -66,6 +69,8 @@ export class Navigation {
                 </slot>
               </li>
             )}
+
+            <slot name="secondary-items" />
```

Comment:
Is the slot should be outside `<ul>` ?  If we have it inside `<ul>`, it's limiting us to ALWAYS add `<li>` before any component we wanted to put there by ourselves. And more documentation needed to make it clear as well. So I suggest to have it outside `<ul>` so people can add whatever component they have in the header or navigation. For example if we have react component called `<my-dropdown>`, then it will be weird if it is rendered as:
`<ul><my-dropdown /></ul>`

Reply:
I agreed, have changed markup to provide a more fitting structure.